components/widgets/icons/camera_icon.jsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+export default class CameraIcon extends React.PureComponent {
```

Comment:
Would you mind changing this to a functional component since it just has a render method? We're trying to start using functional components whenever we can since they let us use hooks and since they're supposed to start having better performance in future versions of React.